src/azure-cli/azure/cli/command_modules/role/tests/latest/test_graph.py

```diff
@@ -272,7 +274,6 @@ def test_graph_group_scenario(self):
             'group': 'deleteme_g',
             'pass': 'Test1234!!'
         }
-        self.recording_processors.append(AADGraphUserReplacer('@' + domain, '@example.com'))
```

Comment:
We'd better still have this line to remove the tenant info from recordings.

Reply:
Add the replay_processor. Have no idea why previous test can pass.

Reply:
updated